*** spatialite.c	2010-11-14 15:59:30.000000000 +0100
--- spatialite-patched.c	2010-12-06 11:08:40.000000000 +0100
***************
*** 81,94 ****
  #include <localcharset.h>
  #endif /* end localcharset */
  #else /* not WINDOWS */
- #ifdef __APPLE__
- #include <iconv.h>
- #include <localcharset.h>
- #else /* not Mac OsX */
  #include <iconv.h>
  #include <langinfo.h>
  #endif
- #endif
  
  #if OMIT_GEOS == 0		/* including GEOS */
  #include <geos_c.h>
--- 81,89 ----
***************
*** 8750,8761 ****
  #if defined(__MINGW32__) || defined(_WIN32)
      return locale_charset ();
  #else /* not MINGW32 - WIN32 */
- #ifdef __APPLE__
-     return locale_charset ();
- #else /* not Mac OsX */
      return nl_langinfo (CODESET);
  #endif
- #endif
  }
  
  GAIAAUX_DECLARE int
--- 8745,8752 ----
